packages/chart/src/Radar.ts

```diff
@@ -0,0 +1,332 @@
+import { INDChart, ITooltip } from "@hpcc-js/api";
+import { SVGWidget, Utility } from "@hpcc-js/common";
+import { hsl as d3Hsl } from "d3-color";
+import { select as d3Select } from "d3-selection";
+
+import "../src/Radar.css";
+export class Radar extends SVGWidget {
+    protected polar2cartesian = (r, theta) => [r * Math.cos(theta), r * Math.sin(theta)];
+    protected cartesian2polar = (x, y) => [Math.atan2(y, x), Math.sqrt(x * x + y * y)];
+    protected degrees2radians = d => d / 57.295779513;
+    protected radians2degrees = r => r * 57.295779513;
+    protected point_in_polygon = (point, vs) => {
+        const x = point[0];
+        const y = point[1];
+        let inside = false;
+        for (let i = 0, j = vs.length - 1; i < vs.length; j = i++) {
+            const xi = vs[i][0];
+            const yi = vs[i][1];
+            const xj = vs[j][0];
+            const yj = vs[j][1];
+            const intersect = ((yi > y) !== (yj > y))
+                && (x < (xj - xi) * (y - yi) / (yj - yi) + xi);
+            if (intersect) inside = !inside;
+        }
+        return inside;
+    }
```

Comment:
For consistency either move the above functions into the class (as static methods if you like) or move them out of the class and simply make them regular functions.

The way it is now each one of the members gets initialized during each Radar instance...